modules/filtering-insights-advisor-recommendations.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="filtering-unnecessary-advisor-recommendations_{context}"]
+= Filtering Insights advisor recommendations
+
+As an OpenShift cluster manager, you can filter the recommendations that are displayed on the recommendations list. By applying filters, you can reduce the number of reported recommendations and concentrate on your highest priority recommendations.
+
+The following procedure demonstrates how to set and remove *Category* filters; however, the procedure is applicable to any of the primary and secondary filters. 
+
+.Prerequisites
+You are logged in to the https://console.redhat.com/openshift[OpenShift Cluster Manager Hybrid Cloud Console].
+
+.Procedure
+. Navigate to the link:https://console.redhat.com/openshift/insights/advisor/recommendations[*Red Hat Hybrid Cloud Console* -> *OpenShift* -> *Advisor recommendations*] page.
+. Using the filter input fields above the list of recommendations, select the *Category* filter type.
+. Expand the right dropdown and check the box next to each category of recommendation you want to view. Leave unnecessary categories unchecked.
+. Optionally, add additional filters to further refine the list.
```

Comment:
```suggestion
. Optional: Add additional filters to further refine the list.
```
Follow [the ISG guidance](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#indicating-optional-and-conditional-steps) for indicating optional steps.

Reply:
Done